Javadoc warnings during MicroProfile 3.1 Release build

The following Javadoc warnings were given during the [MicroProfile 3.1 release build](https://ci.eclipse.org/microprofile/view/Release/job/MicroProfile%20Releases/258/console):

[INFO] [WARNING] /jobs/genie.microprofile/MicroProfile Releases/workspace/target/checkout/target/distro-javadoc-sources/microprofile-jwt-auth-api-1.1-sources/org/eclipse/microprofile/auth/LoginConfig.java:61: warning: no description for @return
[INFO] [WARNING] * @return
[INFO] [WARNING] ^
[INFO] [WARNING] /jobs/genie.microprofile/MicroProfile Releases/workspace/target/checkout/target/distro-javadoc-sources/microprofile-jwt-auth-api-1.1-sources/org/eclipse/microprofile/jwt/JsonWebToken.java:146: warning: no @param for <T>
[INFO] [WARNING] <T> T getClaim(String claimName);
[INFO] [WARNING] ^

--- a/api/pom.xml
+++ b/api/pom.xml
@@ -45,6 +45,7 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.annotation.versioning</artifactId>
             <version>1.1.0</version>
+            <scope>provided</scope>
         </dependency>
     </dependencies>
     <build>

--- a/api/src/main/java/org/eclipse/microprofile/auth/LoginConfig.java
+++ b/api/src/main/java/org/eclipse/microprofile/auth/LoginConfig.java
@@ -58,7 +58,7 @@ public @interface LoginConfig {
     /**
      * The realm name element specifies the realm name to
      use in HTTP Basic authorization.
-     * @return
+     * @return the realm name which may be empty
      */
     public String realmName() default "";
 }

--- a/api/src/main/java/org/eclipse/microprofile/jwt/JsonWebToken.java
+++ b/api/src/main/java/org/eclipse/microprofile/jwt/JsonWebToken.java
@@ -140,6 +140,8 @@ public interface JsonWebToken extends Principal {
 
     /**
      * Access the value of the indicated claim.
+     * 
+     * @param <T> The claim type
      * @param claimName - the name of the claim
      * @return the value of the indicated claim if it exists, null otherwise.
      */

